fe/fe-core/src/main/java/org/apache/doris/system/Backend.java

```diff
@@ -116,8 +119,14 @@ public enum BackendState {
     // additional backendStatus information for BE, display in JSON format
     @SerializedName("backendStatus")
     private BackendStatus backendStatus = new BackendStatus();
-    @SerializedName("tag")
-    private Tag tag = Tag.DEFAULT_BACKEND_TAG;
+    // the locationTag is also saved in tagMap, use a single field here to avoid
+    // creating this everytime we get it.
+    @SerializedName(value = "locationTag", alternate = {"tag"})
+    private Tag locationTag = Tag.DEFAULT_BACKEND_TAG;
```

Comment:
Will we delete this field `locationTag` in the future?
It may be confusing if we keep a special `entry` out of the `tagMap`.

Reply:
It is ok to leave this field here. Because the locationTag is used very frequent.
We don't need to create it every time we get it.
And in future, I will remove `@ SerializedName` annotation of it because it has been saved in `tagMap`